assets/templates/geography-start.tmpl

```diff
@@ -0,0 +1,43 @@
+{{ $lang := .Language }}
+
+<div class="ons-page__container ons-container ons-u-mb-xl">
+    <div class="ons-grid ons-js-toc-container ons-u-ml-no">
+        <div class="ons-grid__col ons-col-8@m ons-u-mb-m">
+            {{ template "partials/breadcrumb" . }}
+        </div>
+
+        <h1 class="ons-grid__col ons-col-8@m ons-u-fs-xxxl ons-u-mb-m">
+            {{ localise "FindFactsAndFiguresEnglandWales" $lang 1 }}
+        </h1>
+
+        <div class="ons-u-bb  ons-u-mb-s">
```

Comment:
The above suggestion causes a problem with `ons-u-bb` but as your design doesn't have it, remove it

Reply:
The border was added to the Figma designs after the image was attached to the ticket.
I've added a screenshot from Figma to the PR.
I've refactored this class onto a different element to resolve the issue with removing the no margin class.